src/routes/session/session-router.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from "express";
+import { SessionEnterpriseGet, SessionGet } from "routes/session/session-get";
+
+export const SessionRouter = Router();
+
+SessionRouter.get(["/enterprise", "/enterprise/*"], SessionEnterpriseGet);
+SessionRouter.get(["/", "/*"], SessionGet);
```

Comment:
why do we need a separate handler when they both do exactly the same?

Reply:
I thought it would be better to separate the routes to make it simple. 
The view is the same in both cases, but the message displayed are different. We can use the same route and pass the _enterprise_ part as a `queryString` or a separate `route parameter`. But that would just add a bit of complexity on the handler, as it would have to display different messages based on the  `queryString` or `params`. 
So instead of doing the conditional handling, I went with this approach.

Reply:
Reverted back to single route. 
Handled by adding query string.